sys/include/architecture.h

```diff
@@ -7,8 +7,7 @@
  */
 
 /**
- * @defgroup    sys_architecture    Platform-independent access to architecture
- *                                  details
+ * @defgroup    sys_architecture    architecture: Platform-independent details
```

Comment:
architecture is not a module

Reply:
This module provides architecture-independent access to architecture details.